src/main/java/org/springframework/social/bitbucket/api/impl/BitBucketTemplate.java

```diff
@@ -54,6 +57,11 @@ public final RepoOperations repoOperations() {
     }
 
     @Override
+    public final GroupsOperations groupsOperations() {
```

Comment:
Correct, disregard my comment (its getting late here, I'm not reading carefully ...) :+1: